vendor/init.bat

```diff
@@ -7,10 +7,11 @@
 :: !!! Use "%CMDER_ROOT%\config\user-profile.cmd" to add your own startup commands
 
 :: Use /v command line arg or set to > 0 for verbose output to aid in debugging.
+endlocal
```

Comment:
Here's the strange point. If I use
```batch
endlocal
set "CMDER_USER_FLAG= "
setlocal
```
the whole `init.bat` won't behave properly.

Reply:
Also I'm not sure if %GIT_INSTALL_ROOT% goes wrong have something to do with this.
I run `echo %GIT_INSTALL_ROOT%` and the output is `!GIT_INSTALL_ROOT!`. And I'm quite confused.